doc/developers/advanced_installation.rst

```diff
@@ -102,6 +103,30 @@ On Unix-like systems, you can simply type ``make`` in the top-level folder to
 build in-place and launch all the tests. Have a look at the ``Makefile`` for
 additional utilities.
 
+Mac OSX
+-------
+
+The default C compiler, Apple-clang, on Mac OSX does not directly support
+OpenMP. The first solution to build scikit-learn is to install another C
+compiler such as gcc or llvm-clang. Another solution is to enable OpenMP
+support on the default Apple-clang.
+
+You first need to install the OpenMP library::
+
+    brew install libomp
+
+Then you need to set the following environment variables::
+
+    export CC=clang
```

Comment:
Should we specify clang explicitly?

```bash
export CC=/usr/bin/clang 
export CXX=/usr/bin/clang++
```

This is to make sure the homebrewed clang is used and not clang from something like anaconda.

Reply:
Yes I thought it was for the travis build, and not for the install doc, but it's been added in #13093